aidial_analytics_realtime/app.py

```diff
@@ -48,10 +37,22 @@ async def startup_event():
     rates_calculator = RatesCalculator()
     app.dependency_overrides[RatesCalculator] = lambda: rates_calculator
 
+    yield
+
+    logger.info("Application shutdown started.")
+    await influx_client.close()
```

Comment:
It might be better to use
```python
with contextlib.closing(influx_client):
   app.dependency_overrides[InfluxWriterAsync] = lambda: influx_writer
   ...
   yield
```
to have close in case of exceptions.

Reply:
`contextlib.closing` expects non-async `close` method, `contextlib.closing` expects async `aclose` method, so neither works.

Used `influx_client` as a context manager instead.